package.json

```diff
@@ -2,67 +2,63 @@
   "name": "dashjs",
   "version": "3.2.2",
   "description": "A reference client implementation for the playback of MPEG DASH via Javascript and compliant browsers.",
-  "main": "build/es5/index.js",
-  "types": "build/typings/index.d.ts",
+  "main": "dist/dash.all.debug.js",
+  "files": [
+    "index.d.ts",
+    "dist",
+    "doc"
+  ],
+  "types": "index.d.ts",
   "scripts": {
-    "build": "npx typescript && grunt",
+    "dev": "webpack --config build/webpack.dev.js --mode development --watch --progress",
```

Comment:
what is the difference to ```npm run start``` except this one does not serve and launch the browser?

Reply:
As you said it does not start the dev server. This mode is useful if you simply want to watch and rebuild your files automatically when project is linked (npm link) into another project in which you integrate dash.js.

Reply:
ah, great. Haven't thought about this. thx